README.md

```diff
@@ -10,132 +10,23 @@ This repo contains the early stages of the Exchange Union Daemon ("xud") which e
-
-```bash
-~/xud/bin $ ./xud --help
-Options:
-  --help                        Show help                              [boolean]
-  --version                     Show version number                    [boolean]
-  --xudir, -x                   Data directory for xud                  [string]
-  --db.database                 SQL database name                       [string]
-  --db.host                     Hostname for SQL database               [string]
-  --db.port                     Port for SQL database                   [number]
-  --db.username                 User for SQL database                   [string]
-  --lnd.certpath                Path to the SSL certificate for lnd     [string]
-  --lnd.disable                 Disable lnd integration                [boolean]
-  --lnd.host                    Host of the lnd gRPC interface          [string]
-  --lnd.macaroonpath            Path of the admin macaroon for lnd      [string]
-  --lnd.port                    Port of the lnd gRPC interface          [number]
-  --p2p.listen                  Listen for incoming peers              [boolean]
-  --p2p.port, -p                Port to listen for incoming peers       [number]
-  --raiden.disable              Disable raiden integration             [boolean]
-  --raiden.port                 Port for raiden REST service            [number]
-  --rpc.port, -r                RPC service port                        [number]
-  --webproxy.disable            Disable web proxy server               [boolean]
-  --webproxy.port               Port for web proxy server               [number]
-```
-
-## Command-Line Interface
-
-Interact with an `xud` process, identified by its `rpc` host and port
-
-```bash
-~/xud/bin $ ./xucli --help
-xucli [command]
-
-Commands:
-  xucli connect <host> [port]               connect to an xu node
-  xucli getinfo                             get general info from the xud node
-  xucli getorders [pair_id] [max_results]   get orders from the order book
-  xucli getpairs                            get order book's available pairs
-  xucli placeorder <pair_id> <price>        place an order
-  <quantity>
-  xucli shutdown                            gracefully shutdown the xud node
-  xucli tokenswap <identifier> <role>       perform a raiden token swap
-  <sending_amount> <sending_token>
-  <receiving_amount> <receiving_token>
-
-Options:
-  --help          Show help                                            [boolean]
-  --version       Show version number                                  [boolean]
-  --rpc.port, -p  The RPC service port                  [number] [default: 8886]
-  --rpc.host, -h  The RPC service hostname       [string] [default: "localhost"]
-```
-
-## Database Initialization
-
-To initialize the database with default testing data, run the following command:
-
-```bash
-npm run db:init
-```
-
-## Documentation
-
-[Read the TypeDoc here](https://exchangeunion.github.io/xud-typedoc/).
+## Questions?
```

Comment:
Maybe call this something like "Development & Testing Discussion"? Also, add a blank line after the heading.